modules/cudn-status-conditions.adoc

```diff
@@ -0,0 +1,84 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cudn-status-conditions_{context}"]
+= ClusterUserDefinedNetwork status condition types
+
+The following tables explain the status conditions types returned for `ClusterUserDefinedNetwork` CRs when describing the resource. These can be used to troubleshoot your deployment.
+
+//The following table is subject to change and will be updated accordingly.
+
+.NetworkCreated condition types
+[cols="2a,2a,3a,6a",options="header"]
+|===
+
+|Condition type
+|Status
+2+|Reason and Message
+
+.3+|`NetworkCreated`
+.3+| `True`
+2+|When `True`, the following reason and message is returned:
+h|Reason
+h|Message
+
+|`NetworkAttachmentDefinitionCreated`
+|'NetworkAttachmentDefinition has been created in following namespaces: [example-namespace-1, example-namespace-2, example-namespace-3]'`
+
+.9+|`NetworkCreated`
+.9+| `False`
+2+|When `False`, one of the following messages is returned:
+h|Reason
+h|Message
+
+|`SyncError`
+|`failed to generate NetworkAttachmentDefinition`
+
+|`SyncError`
+|`failed to update NetworkAttachmentDefinition`
+
+|`SyncError`
+|`primary network already exist in <namespace_name>: <primary_network_name>`
+
+|`SyncError`
+|`failed to create NetworkAttachmentDefinition: create NAD error`
+
+|`SyncError`
+|`foreign NetworkAttachmentDefinition with the desired name already exist`
+
+|`SyncError`
+|`failed to add finalizer to UserDefinedNetwork`
+
+|`NetworkAttachmentDefinitionDeleted`
+|`NetworkAttachmentDefinition is being deleted: [<namespace>/<namespace>]`
```

Comment:
quite sure the second `namespace` is meant to be something different.

Reply:
ahh, looks like it's the <nad_name>: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/9cce8ee50d07c83b6a70e7821dc52e4e67b3c277/go-controller/pkg/clustermanager/userdefinednetwork/controller.go#L697